apps/gazettes-aleph/gazettes-aleph.yml

```diff
@@ -0,0 +1,105 @@
+---
+- hosts:
+    - gazettes_aleph_web
+  become: yes
+  roles:
+    - dokku_bot.ansible_dokku
+
+  vars:
+    app_name: "gazettes-aleph"
+    dokku_daemon_install: false # We install this using a dedicated playbook
+    dokku_plugins:
+      - name: letsencrypt
+        url: https://github.com/dokku/dokku-letsencrypt.git
+      - name: elasticsearch
+        url: https://github.com/dokku/dokku-elasticsearch.git
+    postgres_hostname: postgresql11-prod.cnc362bhpvfe.eu-west-1.rds.amazonaws.com
+    postgres_database: aleph
+    postgres_user: "{{ lookup('passwordstore', 'apps/aleph/gazettes-aleph/{{ env_name }}/POSTGRES subkey=username')}}"
+    postgres_password: "{{ lookup('passwordstore', 'apps/aleph/gazettes-aleph/{{ env_name }}/POSTGRES')}}"
+
+  tasks:
+    - name: "Dokku app exists"
+      dokku_app:
+        app: "{{ app_name }}"
+
+    - name: "set vm.max_map_count to 262144 in sysctl"
+      sysctl: "name={{ item.key }} value={{ item.value }}"
+      with_items:
+        - key: "vm.max_map_count"
+          value: "262144"
+
+    # TODO: Get the following vars exported properly so that the correct image is used
+    # TODO: The following shell command is not idempotent
+    - name: "Dokku elasticsearch"
+      vars:
+        ELASTICSEARCH_IMAGE: "openup/elasticsearch-0.90"
+        ELASTICSEARCH_IMAGE_VERSION: "latest"
```

Comment:
let's try the latest first - so just plain dokku:elasticsearch with its defaults

even if we're not using the latest, the earliest we need to use is 1 which we know works https://github.com/OpenUpSA/docker-elasticsearch

Reply:
@jbothma What do you mean by 1 in this context?